### PR TITLE
Fixes broken link to Configuration page

### LINKED ISSUE #42: "Configuring Fountainhead" link in "Getting Started" guide broken
Hi all!

I was evaluating ember-fountainhead and was reading the guides when I stumbled upon a broken link. On the [Getting Started](https://healthsparq.github.io/ember-fountainhead/guides/getting-started) there's a blue info box which contains a link to [Configuring Fountainhead](https://healthsparq.github.io/guides/configuration). Clicking this link leads me to a GitHub Pages 404.

Hope this helps, keep up the good work!

### PR DESCRIPTION
**Pull request by dennislaumen**
This commit fixes the broken link to the configuration page. This was easily fixed by adding the .md file extension to the link.

Fixes #42

### CODE REVIEW

**Comment by dennislaumen** [comment]
I see I messed up the commit message when I fixed something in the PR. Can I still fix this in this PR or do I have to clone the repo and amend the commit?

**Comment by dennislaumen** [comment]
Fixed the commit message 😃.

**Comment by evrowe** [comment]
Hi @dennislaumen—

I probably should have provided a bit more explanation for this! When you're on the mini-site's [Getting Started page](https://healthsparq.github.io/ember-fountainhead/guides/getting-started), you'll notice there are links in the sidebar. One of those links corresponds to the "Configuring Fountainhead" guide page that we are manually linking to from within the Getting Started Page's content.

That URL is the one you should be aiming to mirror, rather than linking directly to the source markdown file. This ensures that the link on the mini-site takes you to the same fully-rendered page you would expect, rather than to the source markdown file.

File: guides/getting-started.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ module.exports = {
 ```
 
 {{#fountainhead-alert brand='info' dismiss=false}}
-{{fountainhead-svg svgId='info'}}See [Configuring Fountainhead](/guides/configuration)
+{{fountainhead-svg svgId='info'}}See [Configuring Fountainhead](/guides/configuration.md)
```
Comment (evrowe) [inline guides/getting-started.md]: URL should be one of:
- `/ember-fountainhead/guides/configuration`
- `ember-fountainhead/guides/configuration`

Comment (evrowe) [inline guides/getting-started.md]: (note that these URLs probably will not render correctly when running the dummy app locally and will appear to produce a 404; however, they are correct for the live/deployed version of the site).

You could alternatively experiment with a relative URL that works for both environments.

**Comment by dennislaumen** [comment]
@evrowe hmm, that original commit was quite dumb in hindsight, of course that wouldn't work 🤦‍♂️. I quickly applied that fix on the GitHub web interface because you linked to .md file on the GitHub web interface 😃. That's what I get for quickly "fixing" stuff in between work.

I updated the commit and now link to "configuration" which definitely works locally and which I would expect to also work in production looking at the URLs there. Can you confirm whether this is the case or is there a way I can verify this myself?

**Comment by DHedgecock** [comment]
Hi @dennislaumen thanks for noticing this. The issue is actually that the Github docs site is served under the subdomain `/ember-fountainhead/`, so the absolute url `/guides/configuration` works locally but fails when deployed because it is missing the subdomain.

However a relative link `guides/configuration` will also fail because in 'production' this ends up being `/healthsparq/guides/guides/configuration`.

Using a link-to will use the correct `{{rootURL}}` and work for local and prod:
`See {{link-to 'Configuring Fountainhead' 'guides' 'configuration'}}`

Sorry for the runaround.

**Comment by dennislaumen** [comment]
Thanks @DHedgecock, of course the guides are just an Ember app 😃. I hadn't reached that conclusion yet (again, because I'm 100% new to ember-fountainhead). I just pushed a new commit, I hope this is the solution we're looking for here.

**Comment by DHedgecock** [comment]
Will include this with a number of guides updates in v4. Should be out in a few days or less. Thanks @dennislaumen

**Pull request merged**